app/views/aspects/_no_contacts_message.haml

```diff
@@ -7,5 +7,6 @@
   %br
   %br
   = t('.try_adding_some_more_contacts')
-  != t('.or_spotlight', :link => link_to(t(".community_spotlight") , community_spotlight_path))
+  - if current_user.show_community_spotlight_in_stream?
```

Comment:
This is not the good condition, we want to hide this message if the podmin didn't activate community spotlight in diaspora.yml. Besides, #5560 is about the setting in the user panel, not this message, but you are right, both have to be hidden.